symplyphysics/laws/relativistic/relativistic_time_dilation.py

```diff
@@ -1,35 +1,53 @@
+"""
+Relativistic time dilation
+==========================
+
+Time dilation is the difference in elapsed time as measured by two clocks,
+either due to a relative speed between them (special relativity),
+or a difference in gravitational potential between their locations.
```

Comment:
In this law general relativity case is not covered. I suggest to rename the law to reflect that it measures time dilation from speed only.